java/clients/src/test/java/sleeper/clients/admin/TablePropertyReportTest.java

```diff
@@ -72,19 +81,12 @@ void shouldPrintTablePropertyReportWhenChosen() {
                         "{\"rowKeyFields\":[{\"name\":\"key\",\"type\":\"StringType\"}]," +
                         "\"sortKeyFields\":[]," +
                         "\"valueFields\":[{\"name\":\"value\",\"type\":\"StringType\"}]}\n")
-                // Then check properties in sequence to check spacing between them
-                .contains("# The size of the row group in the Parquet files - defaults to the value in the instance properties.\n" +
-                        "sleeper.table.rowgroup.size: 8388608\n" +
-                        "\n" +
-                        "# The size of the page in the Parquet files - defaults to the value in the instance properties.\n" +
-                        "sleeper.table.page.size: 131072\n")
-                // Then check a property with multi-line description
-                .contains("# A file will not be deleted until this number of seconds have passed after it has been marked as\n" +
-                        "# ready for garbage collection. The reason for not deleting files immediately after they have been\n" +
-                        "# marked as ready for garbage collection is that they may still be in use by queries. Defaults to the\n" +
-                        "# value set in the instance properties.\n" +
-                        "sleeper.table.gc.delay.seconds: 600")
-                // Then check a property with multi-line description and custom line breaks
+                // Check property with multi-line description
+                .contains("# The name of the class that defines how compaction jobs should be created.\n" +
+                        "# This should implement sleeper.compaction.strategy.CompactionStrategy. Defaults to the strategy used\n" +
+                        "# by the whole instance (set in the instance properties).\n" +
+                        "sleeper.table.compaction.strategy.class: sleeper.compaction.strategy.impl.SizeRatioCompactionStrategy")
```

Comment:
This property also has a custom line break. We could leave this out and just have the one below.